war/src/test/java/io/lumeer/engine/controller/CollectionMetadataFacadeIntegrationTest.java

```diff
@@ -523,7 +344,7 @@ public void testIsUserCollection() {
    //
    //      String name = "attribute 1";
    //      collectionMetadataFacade.addOrIncrementAttribute(collection, name);
-   //      collectionMetadataFacade.retypeCollectionAttribute(collection, name, collectionMetadataFacade.COLLECTION_ATTRIBUTE_TYPE_INT);
+   //      collectionMetadataFacade.retypeCollectionAttribute(collection, name, collectionMetadataFacade.ATTRIBUTE_TYPE_INT);
```

Comment:
Why is this test commented? It should either be fixed, removed or marked as ignored with a good explanation.